resources/views/components/actions.blade.php

```diff
@@ -31,7 +31,7 @@ class="{{ filled($action->class) ? $action->class : $theme->actions->headerBtnCl
                     @if(strtolower($action->method) !== ('get'))
                         <form target="{{ $action->target }}"
                               action="{{ route($action->route, $parameters) }}"
-                              method="{{ $action->target }}">
+                              method="post">
```

Comment:
It really was wrong, but you can call the target that was passed to the component, what do you think?

```suggestion
                              method="{{ $action->method }}">
```

Reply:
I refer to [this documentation](https://laravel.com/docs/8.x/blade#method-field). It basically says that all requests, but with the ```get``` method, should be wrapped within a ```post``` request.